### ISSUE TITLE
The bot bot working

### ISSUE THREAD

**Issue by Seydouxe** [TrashEmail/TrashEmail#120]
![image](https://user-images.githubusercontent.com/89177400/209531756-3e5d1252-e9d8-42a2-9460-263b8e9d39a0.png)

**Comment by dvdr00t** [comment]
Yes, apparently the service is not maintained any more. I am experiencing the same issue.
